pages/templates/pages/about.html

```diff
@@ -9,9 +9,10 @@
 		<div class="col-xs-12 col-sm-9 col-md-8">
 			<h1 class="text-dark  ">Hawkpost</h1>
 			<div class="text-left text-lightest ">
-			<p>A lot of years ago pigeons were used to send private messages to people. But to send private and confidential information a pigeon is not enough. We upgraded the pigeon for a hawk, it's stronger, faster and will protect better all your messages.</p>
-			<p>We know that sometimes it's difficult to explain how to encrypt things or how to install that program that will protect confidential information. That's why we created "Hawkpost", a place where someone can easily submit and send sensible information for you in a safe way.</p>
+			<p>For a lot of years pigeons were used to send private messages to people. But today, to send private and confidential information a pigeon is not enough. We upgraded the pigeon for a hawk - it's stronger, faster and will better protect all of your messages.</p>
```

Comment:
Just being picky, but I think "For many years" would read better than "For a lot of years"